File: recipes_3e/apache/web_connect/web_connect.py
#!/usr/bin/python

import cookbook

print('''Content-Type: text/html

<html>
<head><title>Python Web Connect Page</title></head>
<body>
''')

conn = cookbook.connect()
print("<p>Connected</p>")
conn.close()
print("<p>Disconnected</p>")

print('''
</body>
</html>
''')

File: recipes_3e/apache/web_connect/web_connect.php
<html>
<head><title>PHP Web Connect Page</title></head>
<body>

<?php
require_once "Cookbook.php";

try
{
  $dbh = Cookbook::connect ();
  print ("<p>Connected</p>");
}
catch (PDOException $e)
{
  print ("<p>Cannot connect to server</p>");
  exit (1);
}
$dbh = NULL;
print ("<p>Disconnected</p>");
?>
